src/model/DBSCAN/dbscan_optimized.py

```diff
@@ -14,7 +14,7 @@
     'model': dbscan,
     'metadata': {
         'name': 'DBSCAN',
-        'abbreviation': 'dbscan',
+        'abbreviation': 'DBS',
```

Comment:
Tror ikke dette er vanlig?

Reply:
Mer relevant for de litt lengere navnene

Reply:
Nei, kanskje jeg skal legge til et felt. For jeg brukte dette i visualiseringen. Trengte korte navn